pkg/cvo/status.go

```diff
@@ -555,6 +563,16 @@ func convertErrorToProgressing(now time.Time, statusFailure error) (reason strin
 	case payload.UpdateEffectReport:
 		return uErr.Reason, uErr.Error(), false
 	case payload.UpdateEffectNone:
+		m := time.Duration(30)
+		// It takes longer to upgrade MCO
+		if uErr.Name == "machine-config" {
+			m = 3 * m
+		}
+		t := payload.COUpdateStartTimesGet(uErr.Name)
```

Comment:
We'll need to handle various cases where multiple operators are updating (see `uErr.Names`), possibly with different durations:

1. More operators are updating, none of them is slow
2. More operators are updating, one of them is slow
2. More operators are updating, some of them are slow
3. More operators are updating, all of them are slow

Reply:
I knew this would not be simple as it is now. 😝

Reply:
:lolsob:

Reply:
Done.